### ISSUE TITLE
Improve library description and details

### ISSUE THREAD

**Issue by edrpls** [gumgum/gumdrops#2]
There are a few improvements we could add to the library details:
- Add license type to generated build package.json
- Copy license.txt to the build on tools/build.js
- Improve library description
- Clarify difference between DS documentation and Storybook

Please comment this ticket with any other improvements for this version.

**Comment by jscottsmith** [comment]
Not related to a release, but we should also add the [storybook link](https://gumdrops.gumgum.com/) next to the Github description: 

![screen shot 2018-03-29 at 4 39 09 pm](https://user-images.githubusercontent.com/7143485/38118525-1c3d30e8-3370-11e8-9294-37ee19ffe05a.png)

**Comment by edrpls** [comment]
@jscottsmith for sure, actually, if it's not tied to the code, would you mind adding it?

**Comment by jscottsmith** [comment]
I can't edit those details, I think I need permissions.

**Comment by edrpls** [comment]
Oh... Nevermind then. I'll see if I can edit them.

**Comment by mnicole** [comment]
i think we need to get permissions from ken or someone... i wanted to add some labels and couldn't.

**Comment by edrpls** [comment]
@mnicole makes sense, I don't even see the options @jscottsmith shared. Paging @kweiner.

**Comment by mnicole** [comment]
also:
- the examples are all using yarn, but clarify or add a note, that you can also use npm
- show an example of a full component in the "using" section. right now we only show how to import a component

**Comment by edrpls** [comment]
uh... Nevermind, I was able to change the website URL, I guess we just need permissions for the labels then.

@mnicole thanks for the suggestions, I'll add it to my original comment.

**Comment by mnicole** [comment]
@kweiner is it possible to add @jscottsmith as a member of 'gumgum' and to give us all more permissions for this repo? thanks!
